docs/index.md

```diff
@@ -1153,7 +1154,7 @@ id | [transactionHash](#transaction-id) | A hash of the transaction that can be
 address | [address](#address) | The address of the account that initiated the transaction.
 sequence | [sequence](#account-sequence-number) | The account sequence number of the transaction for the account that initiated it.
 type | [transactionType](#transaction-types) | The type of the transaction.
-specification | object | A specification that would produce the same outcome as this transaction. The structure of the specification depends on the value of the `type` field (see [Transaction Types](#transaction-types) for details). *Note:* This is **not** necessarily the same as the original specification.
+specification | object | A specification that would produce the same outcome as this transaction, but for payment transactions, without the `destination.amount`. The structure of the specification depends on the value of the `type` field (see [Transaction Types](#transaction-types) for details). *Note:* This is **not** necessarily the same as the original specification.
```

Comment:
The rephrased sentence is a little tough to understand. Here's my suggested change:

> A specification that would produce the same outcome as this transaction. *Exception:* For payment transactions, this omits the `destination.amount` field, to prevent misunderstanding.